src/Persistence.Tests/Yaml/DeserializerInvalidTests.cs

```diff
@@ -10,12 +9,12 @@ namespace Persistence.Tests.Yaml;
 public class DeserializerInvalidTests
 {
     [TestMethod]
-    public void Deserialize_ShouldFail()
+    public void Deserialize_ShouldFailWhenYamlIsInvalid()
     {
         // Arrange
         var deserializer = YamlSerializationFactory.CreateDeserializer();
 
-        var files = Directory.GetFiles(@"_TestData/InvalidYaml", $"*{MsappArchive.YamlFxFileExtension}", SearchOption.AllDirectories);
+        var files = Directory.GetFiles(@"_TestData/InvalidYaml", $"*.fx.yaml", SearchOption.AllDirectories);
```

Comment:
Why do we need to hard code extension here?

Reply:
I removed the `YamlUtils` class along with the `Screen` loading code from msapp.